### PR TITLE
Add Jupyter widget for Griduniverse

### PR DESCRIPTION
**Pull request by MatthewWilkes**
This adds a basic Jupyter widget for Griduniverse and documents
its use through a Jupyter notebook.

### CODE REVIEW

**Comment by MatthewWilkes** [comment]
The following is a sample data file, as needed by the Jupyter notebook:
[6195d89e-c2c0-c65c-8f09-ba0f27143b35-data.zip](https://github.com/Dallinger/Griduniverse/files/1932980/6195d89e-c2c0-c65c-8f09-ba0f27143b35-data.zip)

It should be placed in `dlgr/Griduniverse/data/`

File: dlgr/griduniverse/jupyter.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                chat_template.render(
+                    chat_messages=self.exp.grid.chat_message_history,
+                )
+            )
+            tab_list += [chat_tab]
+
+        tabs = widgets.Tab(children=tab_list)
+        tabs.set_title(0, 'Grid')
+        tabs.set_title(1, 'Scores')
+        tabs.set_title(2, 'Chat')
```
Comment (davisagli) [inline dlgr/griduniverse/jupyter.py]: I think it'd be nice to still include the tab listing configuration that I implemented for the generic widget.

Comment (MatthewWilkes) [reply dlgr/griduniverse/jupyter.py]: Good point, I'll re-add it.

**Comment by MatthewWilkes** [comment]
Closing this in favour of #176

**Pull request closed without merging**